data_import/models.py

```diff
@@ -198,12 +198,22 @@ class NewDataFileAccessLog(models.Model):
     data_file = models.ForeignKey(
         DataFile, related_name="access_logs", on_delete=models.SET_NULL, null=True
     )
+    serialized_data_file = JSONField(default=dict, null=True)
     data_file_key = JSONField(default=dict, null=True)
     aws_url = models.CharField(max_length=400, null=True)
 
+    @property
+    def get_data_file(self):
```

Comment:
As long as the `data_file` field is still present in this model, I'd rather not add this method here